Sources/SpelledPitch/PitchSpeller/Wetherfield/Pitch.Spelling.Category.swift

```diff
@@ -23,7 +23,7 @@ extension Pitch.Spelling {
         /// Category for pitch classes `0` and `5`.
         struct Zero: PitchSpellingCategoryProtocol {
             static var modifiers: ModifierMap {
-                return [.down: .flat, .neutral: .natural, .up: .sharp]
+                return [.down: .doubleFlat, .neutral: .natural, .up: .sharp]
```

Comment:
Ooooh dear, good catch 😮 .

Reply:
Things you find when you think a problem could be _literally anything_ ...